stl/inc/codecvt

```diff
@@ -21,9 +21,9 @@ _STL_DISABLE_CLANG_WARNINGS
 #pragma warning(disable : 4127) // conditional expression is constant
 
 _STD_BEGIN
-#define _LITTLE_FIRST   1
-#define _BIG_FIRST      2
-#define _BYTES_PER_WORD 4
+constexpr int _Little_first   = 1;
+constexpr int _Big_first      = 2;
+constexpr int _Bytes_per_word = 4;

```

Comment:
`_BYTES_PER_WORD` appears to have been completely unused and should be eliminated.